tools/test_configs/RealtekInterface.json

```diff
@@ -8,8 +8,8 @@
             "value" : "new RTWInterface()"
         },
         "connect-statement" : {
-            "help" : "Disabled until EMAC updated",
-            "value" : null
+            "help" : "Must use 'net' variable name, replace WIFI_SSID, WIFI_PASSWORD, WIFI_SECURITY, WIFI_CHANNEL with your WiFi settings",
+            "value" : "(net->wifiInterface()->connect(WIFI_SSID, WIFI_PASSWORD, WIFI_SECURITY, WIFI_CHANNEL)"
```

Comment:
This still has a stray opening `'(` at the front. I can edit this myself - going to need to rebase and tidy this commit.

Reply:
Thank you!